components/calendar/calendar.js

```diff
@@ -544,29 +537,30 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 		const role = this._dialog ? 'dialog' : undefined;
 		return html`
 			<div aria-labelledby="${ifDefined(labelledBy)}" class="${classMap(calendarClasses)}" role="${ifDefined(role)}">
-				<div class="d2l-calendar-title">
-					<d2l-button-icon
-						@click="${this._onPrevMonthButtonClick}"
-						text="${this._computeText(getPrevMonth(this._shownMonth))}"
-						icon="tier1:chevron-left">
-					</d2l-button-icon>
-					<h2 aria-atomic="true" aria-live="polite" class="d2l-heading-4" id="${labelId}">${heading}</h2>
-					<d2l-button-icon
-						@click="${this._onNextMonthButtonClick}"
-						text="${this._computeText(getNextMonth(this._shownMonth))}"
-						icon="tier1:chevron-right">
-					</d2l-button-icon>
+				<div role="application">
```

Comment:
`role="application"` gives us control over the key events, which I believe was most problematic for NVDA + Firefox